libs/core/device-manager/services/usb-devices/usb-devices-mac.helper.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { exec } from "child_process"
+import { PortInfo } from "serialport"
+
+interface DeviceDetails {
+  [key: string]: string | undefined
+  name?: string
+  ProductID?: string
+  VendorID?: string
+  Version?: string
+  SerialNumber?: string
+  Speed?: string
+  Manufacturer?: string
+  LocationID?: string
+  "CurrentAvailable(mA)"?: string
+  "CurrentRequired(mA)"?: string
+  "ExtraOperatingCurrent(mA)"?: string
+}
+
+const getHarmonyMSCDevice = (output: string): DeviceDetails | null => {
+  const devices: Array<DeviceDetails> = []
+  const lines = output.trim().split("\n")
+  let currentDevice: DeviceDetails = {}
+  let currentKey = ""
+
+  lines.forEach((line) => {
+    if (line.trim() && !line.trim().endsWith(":")) {
+      const [keyParts, ...valueParts] = line.trim().split(":")
+      const key = keyParts.replaceAll(" ", "")
+      const value = valueParts.join(":").trim()
+      currentDevice[key] = value
+    } else if (line.trim().endsWith(":")) {
+      if (Object.keys(currentDevice).length) {
+        devices.push({ name: currentKey, ...currentDevice })
+        currentDevice = {}
+      }
+      currentKey = line.trim().slice(0, -1)
+    }
+  })
+
+  if (Object.keys(currentDevice).length) {
+    devices.push({ name: currentKey, ...currentDevice })
+  }
+
+  return (
+    devices.find((device) => {
+      if (device.VendorID && device.ProductID) {
+        return (
+          device.VendorID.includes("3310") && device.ProductID.includes("0103")
```

Comment:
🟢 WDYT?
```suggestion
          device.VendorID.includes(VendorID.MuditaHarmony) && device.ProductID.includes(ProductID.MuditaHarmonyMsc)
```

Reply:
Yes! Forget that we already have this constants in our system!